src/OSPSuite.R/Mapper/SensitivityAnalysisToCoreSensitivityAnalysisMapper.cs

```diff
@@ -53,14 +53,19 @@ public CoreSensitivityAnalysis MapFrom(SensitivityAnalysis sensitivityAnalysis)
          return coreSensitivityAnalysis;
       }
 
-      private IReadOnlyList<string> parameterPathsToVaryFrom(SensitivityAnalysis sensitivityAnalysis)
+      public bool ParameterCanBeUsedForSensitivity(IParameter parameter)
       {
-         if (sensitivityAnalysis.ParameterPaths.Any())
-            return sensitivityAnalysis.ParameterPaths;
```

Comment:
@Yuri05 I have changed the logic here to verify that the parameter selected by the user can in-fact be varied
Otherwise it is potentially possible to specificy parameter that do not exist or should not be CanBeVaried etc...